dataminer-overview/Legal/Commercial_Terms/Terms_partners.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Terms_partners
+---
+
+# Terms for partners
+
+When submitting the partnership application on Skyline Communications' website, by selecting the checkbox, you confirm that you have read, have understood, and agree to be bound by the Partner Program Terms below, including any linked policies. This action constitutes a legally binding acceptance of the terms under Belgian law.
```

Comment:
@michelhofmans, I have rephrased the end of your document as the introduction here. Does this look OK for you?

Reply:
Good, thank you.